src/commands/dao_cmds/apps.js

```diff
@@ -69,34 +75,73 @@ exports.handler = async function({
           })
         },
       },
+      {
+        title: 'Fetching permissionless apps',
+        enabled: () => all,
+        task: async (ctx, task) => {
+          const kernel = new web3.eth.Contract(
+            getContract('@aragon/os', 'Kernel').abi,
+            ctx.daoAddress
+          )
+
+          const events = await kernel.getPastEvents('NewAppProxy', {
+            fromBlock: 0,
```

Comment:
This could take a while on mainnet, we could instead use the Kernel's initialization block (`kernel.getInitializationBlock()`) as the from block